main.py

```diff
@@ -35,8 +35,8 @@ def __init__(self):
         # Connect threading signals
         self.add_frame.connect(self.setup_videoframe)
         self.fail_add_stream.connect(self.on_fail_add_stream)
-
-        self.vlc_instance = vlc.Instance("--no-xlib")
+        # Opengl performs better on windows, which is odd
+        self.vlc_instance = vlc.Instance("--no-xlib --vout=opengl --avcodec-hw=any --avcodec-threads=0")
```

Comment:
Yep no performance gain after testing again. Which also makes sense as we are using callbacks (they block hardware decoding).